scripts/3-export-libtensorflow-cc.sh

```diff
@@ -26,10 +26,10 @@ CONTAINER_FILE="libtensorflow-cc_${TF_VERSION}.deb"
 EXPORT_FILE="libtensorflow-cc_${TF_VERSION}${GPU_POSTFIX}.deb"
 STAGE="deb-package"
 
-echo "Building ${IMAGE_LIBTENSORFLOW_CC} ... "
-docker build --build-arg TF_VERSION=${TF_VERSION} --build-arg JOBS=${JOBS} --build-arg GPU_POSTFIX=${GPU_POSTFIX} --target ${STAGE} -t ${IMAGE_LIBTENSORFLOW_CC} ${DOCKER_DIR} | tee ${LOG_FILE}
+echo "Building ${IMAGE_LIBTENSORFLOW_CC_ARCH} ... "
+docker build --build-arg TARGETARCH=$ARCH --build-arg TF_VERSION=${TF_VERSION} --build-arg JOBS=${JOBS} --build-arg GPU_POSTFIX=${GPU_POSTFIX} --target ${STAGE} -t ${IMAGE_LIBTENSORFLOW_CC_ARCH} ${DOCKER_DIR} | tee ${LOG_FILE}
 
 echo "Exporting to $(realpath ${EXPORT_DIR})/${EXPORT_FILE} ... "
-TMP_CONTAINER=$(docker create ${IMAGE_LIBTENSORFLOW_CC})
+TMP_CONTAINER=$(docker create ${IMAGE_LIBTENSORFLOW_CC_ARCH})
 docker cp ${TMP_CONTAINER}:/${EXPORT_FILE} ${EXPORT_DIR}/
```

Comment:
Should `EXPORT_FILE` already include the arch-postfix?

Reply:
Added in [c9b976a](https://github.com/ika-rwth-aachen/libtensorflow_cc/pull/2/commits/c9b976ac9715eccd3b07c0b4ec2c71e1b106208d).